ckan/logic/action/create.py

```diff
@@ -834,7 +834,8 @@ def organization_create(context, data_dict):
 
     '''
     # wrapper for creating organizations
-    data_dict['type'] = 'organization'
+    if 'type' not in data_dict:
+        data_dict['type'] = 'organization'
```

Comment:
There's [`dict.setdefault`](https://docs.python.org/2/library/stdtypes.html#dict.setdefault) for this pattern.